lib/site_prism/element_container.rb

```diff
@@ -24,7 +24,7 @@ def elements(collection_name, *find_args)
     alias collection elements
 
     def expected_elements(*elements)
-      @expected_items = elements.map(&:to_s)
+      @expected_items = elements
```

Comment:
@luke-hill This addresses your comment in #208. You're correct, the `to_s` is unnecessary.